mypy/main.py

```diff
@@ -730,6 +731,11 @@ def add_invertible_flag(flag: str,
     if special_opts.no_executable:
         options.python_executable = None
 
+    # Paths listed in the config file will be ignored if any paths are passed on
+    # the command line.
+    if not special_opts.files and options.files:
```

Comment:
I'd write this as `if options.files and not special_opts.files:` -- just because in English the priority of `not` vs. `and` is not so clear.

Reply:
Yep, that's how I did it the first time and then decided to flip it. I'll change it back.